loadfile/loadfile_test.go

```diff
@@ -0,0 +1,102 @@
+package loadfile_test
+
+import (
+	"go.arcalot.io/assert"
+	"go.flow.arcalot.io/engine/loadfile"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// This tests the functional behavior of LoadContext when it is given
+// a directory of files that cover more than the directory file node
+// type 'd' (as seen in *nix systems). Specifically, this test adds
+// symbolic link files for a regular file and a directory file. The
+// engine should only attempt to read files of a type that the os can
+// read (i.e. not throw an error on a call to os.ReadFile()), and
+// disregard (not throw an error) files with a type it cannot read.
+func TestLoadContext(t *testing.T) {
+	testdir := "/tmp/loadfile-test"
+	// cleanup directory even if it's there
+	_ = os.RemoveAll(testdir)
```

Comment:
I think you can omit the `_ = ` and just call `os.RemoveAll()` like a subroutine if you don't care about the value that it returns.

Reply:
Maybe. Sometimes the linter doesn't like that we're not using the value. This format explicitly shows that we're aware of the fact that we're throwing it away.

Reply:
There are lots of calls made to functions which return values in which the returned value is not used.  Go itself does not object in these cases (I cannot comment upon the linter's behavior).

> This format explicitly shows that we're aware of the fact that we're throwing it away.

True.  The question is, is that valuable enough to justify the visual clutter.  For instance, apparently this call was "weird enough" to justify having a code comment attached...and yet the comment doesn't mention that the returned value is being discarded or why (although the "even if it's there" _hints_ at it).  It would be better if the comment were expanded and the assignment were omitted...if the linter will permit that.  And, if the linter won't, it would be good to understand _why_ the linter thinks that it is necessary to explicitly capture a value that we intend to ignore.